Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.py

```diff
@@ -6637,6 +7738,318 @@ def certificate_intermediate_group_list_command(
+        source_address_type=args.get("source_address_type"),
+        ip_range=args.get("ip_range"),
+        ip_type=args.get("ip_type"),
+        ip=args.get("ip"),
+        source_domain_type=args.get("source_domain_type"),
+        source_domain=args.get("source_domain"),
+    )
+    command_results = generate_simple_command_results(
+        object_type=OutputTitle.URL_ACCESS_RULE_CONDITION.value,
+        object_name=condition_id,
+        action=OutputTitle.UPDATED.value,
+        response=response,
+    )
+
+    return command_results
+
+
+def url_access_rule_condition_delete_command(
+    client: Client, args: dict[str, Any]
+) -> CommandResults:
+    """Delete an URL access rule condition.
+
+    Args:
+        client (Client): FortiwebVM API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+
+    group_name = args["group_name"]
+    condition_id = args["condition_id"]
+    response = client.url_access_rule_condition_delete_request(group_name, condition_id)
+    command_results = generate_simple_command_results(
+        object_type=OutputTitle.URL_ACCESS_RULE_CONDITION.value,
+        object_name=condition_id,
+        action=OutputTitle.DELETED.value,
+        response=response,
+    )
+    return command_results
+
+
+def url_access_rule_condition_list_command(
+    client: Client, args: dict[str, Any]
+) -> CommandResults:
+    """List URL access rule conditions.
+
+    Args:
+        client (Client): FortiwebVM API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    group_name = args["group_name"]
+    condition_id = args.get("condition_id")
+    response = client.url_access_rule_condition_list_request(
+        group_name, condition_id=condition_id
+    )
+    parsed_data, pagination_message, _formatted_response = list_response_handler(
+        client,
+        response,
+        client.parser.parse_url_access_rule_condition,
+        args,
+        condition_id,
+        sub_object_key='id'
+    )
+    outputs = {"id": group_name, "Condition": parsed_data}
+    headers = client.parser.create_output_headers(
+        client.version,
+        [
+            "id",
+            "url_type",
+            "url_pattern",
+            "meet_this_condition_if",
+            "source_address_type",
+            "ip_range",
+            "domian_type",
+            "domain",
+            "source_domain_type",
+            "source_domain",
+        ],
+        [],
+        [],
+    )
+    readable_output = tableToMarkdown(
+        name=OutputTitle.URL_ACCESS_RULE_CONDITION.value,
+        metadata=pagination_message,
+        t=outputs["Condition"],
+        headers=headers,
+        headerTransform=string_to_table_header,
+    )
+    command_results = CommandResults(
```

Comment:
There is inconsistency on when we return CommandResults that was created in the method and when we reutnr CommandResults that was created using the helper functions.
Please select one way in order for the code to be consistent.

Reply:
In some cases, the command results is very simple (only ID) and I use the helper command. In cases where the command results has large data I handle them without a helper command